src/ensembl/src/content/app/browser/track-panel/TrackPanel.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import { TrackPanel, TrackPanelProps } from './TrackPanel';
+import TrackPanelBar from './track-panel-bar/TrackPanelBar';
+import TrackPanelList from './track-panel-list/TrackPanelList';
+import TrackPanelModal from './track-panel-modal/TrackPanelModal';
+import Drawer from '../drawer/Drawer';
+
+import { createEnsObject } from 'tests/fixtures/ens-object';
+import { BreakpointWidth } from 'src/global/globalConfig';
+import { TrackSet } from './trackPanelConfig';
+import { createGenomeCategories } from 'tests/fixtures/genomes';
+import { createTrackStates } from 'tests/fixtures/track-panel';
+import configureStore from 'src/store';
+
+describe('<TrackPanel />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const defaultProps: TrackPanelProps = {
+    activeGenomeId: null,
+    breakpointWidth: BreakpointWidth.LARGE,
+    browserActivated: false,
+    isDrawerOpened: false,
+    activeEnsObject: null,
+    isTrackPanelModalOpened: false,
+    isTrackPanelOpened: true,
+    selectedTrackPanelTab: TrackSet.GENOMIC,
+    genomeTrackCategories: createGenomeCategories(),
+    trackStates: createTrackStates()
+  };
+
+  const store = configureStore();
+
+  const wrappingComponent = (props: any) => (
+    <Provider store={store}>{props.children}</Provider>
```

Comment:
Or mock the components that TrackPanel uses so that we don't need to wrap it in redux provider?

Reply:
Thanks! That works like a charm.